release-notes.txt

```diff
@@ -1,3 +1,519 @@
+  for examples/minimal on nrf52dk
+- many more modules and drivers have Kconfig support
+- the semtech-loramac package now supports sx126x/llcc68 radios
+- 8 new boards, 6 new drivers, 8 packages upgraded
+
+We've also mostly finished our migration from mailing lists to forum.riot-os.org.
+
+375 pull requests, composed of 992 commits, have been merged since the
+last release, and 16 issues have been solved. 56 people contributed with
+code in 103 days. 2564 files have been touched with 191220 (+) insertions and
+91400 deletions (-).
+
+
+Notations used below:
+=====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Core
+----
++ clist: added clist_is_empty()
++ xfa: initial support for XFA's (cross file arrays)
+
+System Libraries
+----------------
+* clif: fix attribute parsing
+* shell: allow defining shell commands using XFA
++ dbgpin: module for debugging and profiling
++ ztimer: add ztimer_is_set()
++ ztimer: add ZTIMER_SEC
++ ztimer: add periph_ptp backend
+
+Networking
+----------
+
+* cord: Discovery patch altered to reflect changes in the underlying Resource Directory draft
+* lwip: hook up sam0_eth support
++ congure: framework to implement congestion control and use it in a reusable manner with
+  a number of protocols
++ netstats: L1/L2 per neighbor statistics
++ sock DTLS: allow using multiple credentials
+
+Packages
+--------
+
+* emlearn: bump version to 0.11.6
+* libfixmath: bump version to latest master
+* nanocbor: bump version
+* nanopb: bump version to 0.4.5
+* fatfs: bump version to r0.14a
+* littlefs2: Bump to v2.4
+* monocypher: bump to version 3.1.2
+* nimble: migrate to ztimer, allowing low power operation
+* openthread: bump version to 20191113
+* wakaama: add patch to cast macro to time_t
+
+Boards
+------
+
+* particle-mesh boards: Use USB serial as stdio by default
+* nucleo-f303re: I2C pins remapped to match the data sheet
+* nucleo-f722ze: Add periph_can support
+* native: compile with `-Og` (was `-O0`)
++ boards/nucleo-f767zi: Add periph_can support
++ New boards:
+  + atmega328p-xplained-mini
+  + atxmega-a1u-xpro
+  + atxmega-a3bu-xplained
+  + BastWAN
+  + Microbit v2
+  + samd20-xpro
+  + TI CC1350 LAUNCHXL
+  + Zigduino
+
+CPU
+---
+
++ Initial support for ATxmega family of 8-bit AVR CPUs
++ kinetis: enable floating point support
+* nrf5x: enhance external HF clock source handling to allow for substantial energy
+  savings
++ nrf52: add periph_uart_non_blocking support
++ stm32f7: added ADC support
++ sam0_common: add rtc_get_tamper_event()
++ qn908x: add ADC support
+
+Device Drivers
+--------------
+
++ new drivers:
+  + sensirion SGP30 gas sensor
+  + Amphenol sm-pwm-01c dust sensor
+  + Si705x sensors
+  + TI BQ2429x charger and power management ICs
+  + Netdev driver for nrf24l01p
+  + sx126x/llcc68 radios
+  + touch_dev & display_dev auto-initialization
```

Comment:
This is not a new driver, so should not be part of the "new drivers" subsection (also added other stuff):

```suggestion
 * touch_dev & display_dev auto-initialization
 * driver/bmx280: increase accuracy of SAUL pressure readings
 * driver_lis2dh12: functionality extension
 * drivers/cc110x: add hook cc1xxx_eui_get()
 * drivers/ds3231: add alarm support
 * drivers/at86rf2xx: add missing handler for TRX24_RX_START interrupt
```